t04_refactored.py

```diff
@@ -71,6 +72,7 @@ def kill_if_dead(dead):
     if dead:
         quit()
 
+
```

Comment:
Changes should not have been made outside of your function. Please revert these.